packages/types-algosdk/index.d.ts

```diff
@@ -560,3 +561,96 @@ export interface AccountInfo {
   'created-apps': any;
   'created-assets': any;
 }
+
+export interface TxnEncodedObj {
+  // common fields
+  // https://developer.algorand.org/docs/reference/transactions/#common-fields-header-and-type
+  fee: number
+  fv: number
+  lv: number
+  note: Buffer
+  snd: Buffer
+  type: string
+  gen: string
+  gh: Buffer
+  rekey: Buffer
+  lx: Buffer
+  grp: Buffer
```

Comment:
ditto, I'm pretty sure we should use `Uint8Array` here, and `Uint8Array[]` instead of `Buffer[]`

Reply:
Encoded types were using `Buffer` everywhere instead of `Uint8Array` (eg. `note` is represented as `Uint8Array` in sdk object, but after `get_obj_for_encoding()` it is represented as a `buffer`). Attaching info.

- In SDK (you can check `note` is represented as a buffer)
![image](https://user-images.githubusercontent.com/33264364/101960720-fc5bc680-3c2d-11eb-892d-b621d0e9d3d6.png)
https://github.com/algorand/js-algorand-sdk/blob/develop/src/transaction.js#L162

- Also tried logging the encoded txn object myself:
![image](https://user-images.githubusercontent.com/33264364/101960825-4644ac80-3c2e-11eb-863a-11250f0fb13c.png)

Reply:
ok, thanks.